app/Http/Controllers/Admin/LocationController.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Models\Location;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class LocationController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $locations = Location::all();
+
+        return view('admin.locations.list', [
+            'locations' => $locations
+        ]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('admin.locations.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        // Validate input data
+        $this->validate($request,[
+            'name' => 'required|unique:locations',
+            'city' => 'required|numeric',
+        ]);
+
+        // Saving new location
+        $location = new Location();
+        $location->name = $request->input('name');
+        $location->city_id = $request->input('city');
+        $location->save();
```

Comment:
50,51,52,53 kunnen in een lijn Location::create($request->all()); Moet je wel de fillable zetten op de model